_posts/2024-04-29-back-from-devoxx.adoc

```diff
@@ -0,0 +1,101 @@
+These insights will help you to improve your Keycloak deployments as well as design
+and test your own applications so they can withstand high load and site failures.
+
+[caption="Keycloak",link=/assets/images/blog/2024-devoxx-france/keycloak2.jpg]
+image::/assets/images/blog/2024-devoxx-france/keycloak2.jpg[Keycloak]
+
+==== Infinispan
+During this presentation, Alexander Schwartz and Ryan Emerson delve into the latest
+version of Keycloak and the High Availability architecture that facilitates deploying
+two sites with Keycloak.
+
+[caption="Keycloak",link=/assets/images/blog/2024-devoxx-france/keycloak1.jpg]
+image::/assets/images/blog/2024-devoxx-france/keycloak1.jpg[Keycloak]
+
+One key component enabling this architecture is the
+Infinispan Server, with its Cross-Site replication capabilities.
+
+Initially, this capability is realized through Active-Passive replication,
+but the roadmap includes plans for Active-Active replication.
+
+[caption="Keycloak HA Architecture",link=/assets/images/blog/2024-devoxx-france/keycloak-infinispan.png]
+image::/assets/images/blog/2024-devoxx-france/keycloak-infinispan.png[Keycloak HA Architecture]
+
+
+
+=== Hibernate Search 7: New Features and Practical Cases of Java Full-Text and Vector Search
+
+==== Abstract
+Hibernate Search 7.0 was released in late 2023, and version 7.1 is already available.
+Discover all the new features of this latest version and practical use cases such as
+vectorial search (kNN), which enables the implementation of textual, audio, or
+image-based machine learning search. This conference will guide you through the
+underlying changes, demonstrate concrete use cases, and provide practical tips to make
+the most of the latest version of this essential library for data persistence and
+full-text search in the Java ecosystem.
+
+
+[caption="HIbernate Search",link=/assets/images/blog/2024-devoxx-france/hibernateSearch1.jpg]
+image::/assets/images/blog/2024-devoxx-france/hibernateSearch1.jpg[Hibernate Search]
+
+==== Infinispan
+
+In the talk, Infinispan was mentioned to highlight that Hibernate Search can function as
+a standalone component in a Java Application, independent of Hibernate ORM.
+Infinispan utilizes the Hibernate Search library to implement Full-Text search over
+cache values. Moreover, since Infinispan 15.0, it can be used as an embedding store to
+execute vector search with Langchain and Langchain4j.
+
+
+[caption="Hibernate Search",link=/assets/images/blog/2024-devoxx-france/hibernateSearch2.jpg]
+image::/assets/images/blog/2024-devoxx-france/hibernateSearch2.jpg[Hibernate Search]
+
+=== Les Cast Codeurs Podcast
+
+Mentioning here that Infinispan is indirectly part of Devoxx France and the Les Cast Codeurs podcast too.
```

Comment:
I'm not sure what you're trying to say here?

Reply:
that i'm part of the podcast and devoxx france org

Reply:
I'm the writer of the article, so I don't want o talk about me in 3rd person xDDD

Reply:
We are not using 3rd person in all the article, are we ? I use "we" or Infinispan

Reply:
Maybe I'm too used to technical docs being written in 3rd person :sleepy: . It's just a blog, so let's keep it as is so it's more personal :+1: